.travis.yml

```diff
@@ -24,12 +24,15 @@ addons:
 
 before_install:
   - .travis-scripts/install.sh
-
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then .travis-scripts/osx-opencv-install.sh; fi
 # Only do an assemble when we aren't building a pull request
 install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && ./gradlew :ui:assemble --stacktrace || ./gradlew --stacktrace '
 
 script:
+  - if [[ "TRAVIS_OS_NAME" == "osx" ]]
```

Comment:
@SamCarlberg Do you have an idea as to why Travis is not performing this and the next two lines on osx?